### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by jaytmiller**

### CODE REVIEW

**Comment by jaytmiller** [comment]
That's all I can think of for now.   Let me know if you see anything else.

File: README.rst [hunk 1]
```diff
@@ -523,11 +602,42 @@ configuration script is passed as a 4th generally defaulted parameter to caldp-p
...
-The CALDP repo is set up for Travis via github checkins.   Whenever you do a PR to spacetelescope/caldp,
-Travis will automatically run CI tests for CALDP.
+  $ tox -e flake8
+  $ tox -e bandit
+  $ tox -e black
+
+Initial runs of tox are slow until most the environment is built and cached;
+afterwards CALDP package updates are relatively quick.   The tox cache will be
+treated as part of the Docker build context and uploaded to Docker whenever
+it changes prior to the next build;
```
Comment (bhayden53) [inline README.rst:628]: we should maybe add a .dockerignore

**Review by bhayden53 — APPROVED** [review]
great additions, thanks!

File: README.rst (hunk 1 above)
Comment (jaytmiller) [reply README.rst:628]: Great idea,  coming up!

Comment (jaytmiller) [reply README.rst:628]: Added .dockerignore to this PR.

**Pull request merged**